packages/app-content-pages/src/screens/Publications/PublicationsContainer.js

```diff
@@ -10,7 +10,8 @@ import Publications from './Publications'
 counterpart.registerTranslations('en', en)
 
 function PublicationsContainer(props) {
-  const { error, publicationsData } = props
+  const error = props.error
+  const publicationsData = props.publicationsData || []
```

Comment:
I somehow missed that this is a React component. You can set default values for props for the component in `PublicationsContainer.defaultProps`

Reply:
Perfect :) will do